2025-01-31 21:44:05,463:DEBUG:certbot._internal.main:certbot version: 3.1.0
2025-01-31 21:44:05,463:DEBUG:certbot._internal.main:Location of certbot entry point: /usr/local/bin/certbot
2025-01-31 21:44:05,463:DEBUG:certbot._internal.main:Arguments: ['--webroot', '--webroot-path=/var/www/certbot', '--email', 'karimmaroous@gmail.com', '--agree-tos', '--no-eff-email', '--staging', '-d', 'opsforall.net']
2025-01-31 21:44:05,463:DEBUG:certbot._internal.main:Discovered plugins: PluginsRegistry(PluginEntryPoint#manual,PluginEntryPoint#null,PluginEntryPoint#standalone,PluginEntryPoint#webroot)
2025-01-31 21:44:05,489:DEBUG:certbot._internal.log:Root logging level set at 30
2025-01-31 21:44:05,493:DEBUG:certbot._internal.plugins.selection:Requested authenticator webroot and installer None
2025-01-31 21:44:05,493:DEBUG:certbot._internal.plugins.selection:Single candidate plugin: * webroot
Description: Saves the necessary validation files to a .well-known/acme-challenge/ directory within the nominated webroot path. A separate HTTP server must be running and serving files from the webroot path. HTTP challenge only (wildcards not supported).
Interfaces: Authenticator, Plugin
Entry point: EntryPoint(name='webroot', value='certbot._internal.plugins.webroot:Authenticator', group='certbot.plugins')
Initialized: <certbot._internal.plugins.webroot.Authenticator object at 0x75b4d0453a70>
Prep: True
2025-01-31 21:44:05,495:DEBUG:certbot._internal.plugins.selection:Selected authenticator <certbot._internal.plugins.webroot.Authenticator object at 0x75b4d0453a70> and installer None
2025-01-31 21:44:05,495:INFO:certbot._internal.plugins.selection:Plugins selected: Authenticator webroot, Installer None
2025-01-31 21:44:05,598:DEBUG:certbot._internal.main:Picked account: <Account(RegistrationResource(body=Registration(key=None, contact=(), agreement=None, status=None, terms_of_service_agreed=None, only_return_existing=None, external_account_binding=None), uri='https://acme-staging-v02.api.letsencrypt.org/acme/acct/182784314', new_authzr_uri=None, terms_of_service=None), 5c0ea7419c19db18f607881e60bb62df, Meta(creation_dt=datetime.datetime(2025, 1, 31, 21, 18, 24, tzinfo=datetime.timezone.utc), creation_host='f5a16ff59e49', register_to_eff=None))>
2025-01-31 21:44:05,600:DEBUG:acme.client:Sending GET request to https://acme-staging-v02.api.letsencrypt.org/directory.
2025-01-31 21:44:05,605:DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): acme-staging-v02.api.letsencrypt.org:443
2025-01-31 21:44:05,784:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "GET /directory HTTP/1.1" 200 974
2025-01-31 21:44:05,785:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 31 Jan 2025 21:44:05 GMT
Content-Type: application/json
Content-Length: 974
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "kPEdtinpZ7o": "https://community.letsencrypt.org/t/adding-random-entries-to-the-directory/33417",
  "keyChange": "https://acme-staging-v02.api.letsencrypt.org/acme/key-change",
  "meta": {
    "caaIdentities": [
      "letsencrypt.org"
    ],
    "profiles": {
      "classic": "The same profile you're accustomed to",
      "tlsserver": "https://letsencrypt.org/2025/01/09/acme-profiles/"
    },
    "termsOfService": "https://letsencrypt.org/documents/LE-SA-v1.4-April-3-2024.pdf",
    "website": "https://letsencrypt.org/docs/staging-environment/"
  },
  "newAccount": "https://acme-staging-v02.api.letsencrypt.org/acme/new-acct",
  "newNonce": "https://acme-staging-v02.api.letsencrypt.org/acme/new-nonce",
  "newOrder": "https://acme-staging-v02.api.letsencrypt.org/acme/new-order",
  "renewalInfo": "https://acme-staging-v02.api.letsencrypt.org/draft-ietf-acme-ari-03/renewalInfo",
  "revokeCert": "https://acme-staging-v02.api.letsencrypt.org/acme/revoke-cert"
}
2025-01-31 21:44:05,786:DEBUG:certbot._internal.display.obj:Notifying user: Requesting a certificate for opsforall.net
2025-01-31 21:44:05,792:DEBUG:acme.client:Requesting fresh nonce
2025-01-31 21:44:05,793:DEBUG:acme.client:Sending HEAD request to https://acme-staging-v02.api.letsencrypt.org/acme/new-nonce.
2025-01-31 21:44:05,852:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "HEAD /acme/new-nonce HTTP/1.1" 200 0
2025-01-31 21:44:05,854:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 31 Jan 2025 21:44:05 GMT
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: N9Ep_yiTTQUYV2W2g86ji8REiTpeI0d8gxFVunbcGN4X5MkM7h4
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800


2025-01-31 21:44:05,854:DEBUG:acme.client:Storing nonce: N9Ep_yiTTQUYV2W2g86ji8REiTpeI0d8gxFVunbcGN4X5MkM7h4
2025-01-31 21:44:05,855:DEBUG:acme.client:JWS payload:
b'{\n  "identifiers": [\n    {\n      "type": "dns",\n      "value": "opsforall.net"\n    }\n  ]\n}'
2025-01-31 21:44:05,861:DEBUG:acme.client:Sending POST request to https://acme-staging-v02.api.letsencrypt.org/acme/new-order:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS1zdGFnaW5nLXYwMi5hcGkubGV0c2VuY3J5cHQub3JnL2FjbWUvYWNjdC8xODI3ODQzMTQiLCAibm9uY2UiOiAiTjlFcF95aVRUUVVZVjJXMmc4NmppOFJFaVRwZUkwZDhneEZWdW5iY0dONFg1TWtNN2g0IiwgInVybCI6ICJodHRwczovL2FjbWUtc3RhZ2luZy12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL25ldy1vcmRlciJ9",
  "signature": "FvBpxtNFwe0hD7-7gMl0x6U_kAulZOWBraxWBL8DF02uZ8IzVHd4avXEvjgOYHFLUoP0zGvVDPbeUcK_uKG-6qxK2JoEKZXxof5n0OJIvKjncrPaBjgXf7RJLYrrSNUW3EEwiErNzWUWp3KURBH3vasRBcd3kjkd1XA9-JxHYmXLoBy3eZLxfbh05FvHgcr4ciX_pJZmA2NIO9wqLcu7G8Mnb0wAVRel8Nq4lHvq1EJbqLwhKo-FB1OVUBvS5lXdrrtbNNzKAKGo8-WZKMJheXfLKiZBJ1Gt-qAdsZfuhFtyo3Z6aeJGsvvjCVuCA48A5EJOFWUf1tQfWdW8tAOyPg",
  "payload": "ewogICJpZGVudGlmaWVycyI6IFsKICAgIHsKICAgICAgInR5cGUiOiAiZG5zIiwKICAgICAgInZhbHVlIjogIm9wc2ZvcmFsbC5uZXQiCiAgICB9CiAgXQp9"
}
2025-01-31 21:44:05,951:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "POST /acme/new-order HTTP/1.1" 201 359
2025-01-31 21:44:05,952:DEBUG:acme.client:Received response:
HTTP 201
Server: nginx
Date: Fri, 31 Jan 2025 21:44:05 GMT
Content-Type: application/json
Content-Length: 359
Connection: keep-alive
Boulder-Requester: 182784314
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index"
Location: https://acme-staging-v02.api.letsencrypt.org/acme/order/182784314/22333524044
Replay-Nonce: xMYr_xatYSF5ZxNtx9XnhvPZ3KhPYk80bHI-4Tfn9MtByQ7lFw8
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "status": "pending",
  "expires": "2025-02-07T21:44:05Z",
  "identifiers": [
    {
      "type": "dns",
      "value": "opsforall.net"
    }
  ],
  "authorizations": [
    "https://acme-staging-v02.api.letsencrypt.org/acme/authz/182784314/15902281074"
  ],
  "finalize": "https://acme-staging-v02.api.letsencrypt.org/acme/finalize/182784314/22333524044"
}
2025-01-31 21:44:05,952:DEBUG:acme.client:Storing nonce: xMYr_xatYSF5ZxNtx9XnhvPZ3KhPYk80bHI-4Tfn9MtByQ7lFw8
2025-01-31 21:44:05,952:DEBUG:acme.client:JWS payload:
b''
2025-01-31 21:44:05,953:DEBUG:acme.client:Sending POST request to https://acme-staging-v02.api.letsencrypt.org/acme/authz/182784314/15902281074:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS1zdGFnaW5nLXYwMi5hcGkubGV0c2VuY3J5cHQub3JnL2FjbWUvYWNjdC8xODI3ODQzMTQiLCAibm9uY2UiOiAieE1Zcl94YXRZU0Y1WnhOdHg5WG5odlBaM0toUFlrODBiSEktNFRmbjlNdEJ5UTdsRnc4IiwgInVybCI6ICJodHRwczovL2FjbWUtc3RhZ2luZy12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LzE4Mjc4NDMxNC8xNTkwMjI4MTA3NCJ9",
  "signature": "VMsG-s39MP96IUNG2mvi7Yyaej8Aq2UYbK5f6cf-9v8fth5pyUTzNgN6Q5-kYiOZi9VV90peW5svwkmXsVtdI6mc67CckcssBuXmM1lurPJfCXE7WCZUpqBKS2m3lbYZv5L8tdmFZCiphUGbL72De_wgp3RmfvyAjkMl8M5Z7qpl2W1WKuMlO4lrUbGoApFDQa8Pm6I0j9-RFgpYH8pd7k8yAqkLe1zqFIwoPZMSYYD9XmESKtzHGDeS4g6PzTz0rTBeElCUrhhWXaxQcrAVCa-xrGkp73zvYy0n0EwWtjaLgKE3YRsuUxdSXYWR4Fvu_ifLSGFAzacUdrKbUzYSaA",
  "payload": ""
}
2025-01-31 21:44:06,021:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "POST /acme/authz/182784314/15902281074 HTTP/1.1" 200 839
2025-01-31 21:44:06,022:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 31 Jan 2025 21:44:05 GMT
Content-Type: application/json
Content-Length: 839
Connection: keep-alive
Boulder-Requester: 182784314
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: N9Ep_yiT7kj5TJ4m_kZOfKegEOLPuA5ikXYPQ12Te_B-x3B9kS0
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "opsforall.net"
  },
  "status": "pending",
  "expires": "2025-02-07T21:44:05Z",
  "challenges": [
    {
      "type": "tls-alpn-01",
      "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902281074/GKAXbQ",
      "status": "pending",
      "token": "0c0H_DprVNJOEoPqlNXVneWSM-zzr1eGnzPd2nn7VN0"
    },
    {
      "type": "dns-01",
      "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902281074/FbaGHg",
      "status": "pending",
      "token": "0c0H_DprVNJOEoPqlNXVneWSM-zzr1eGnzPd2nn7VN0"
    },
    {
      "type": "http-01",
      "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902281074/dZlnlA",
      "status": "pending",
      "token": "0c0H_DprVNJOEoPqlNXVneWSM-zzr1eGnzPd2nn7VN0"
    }
  ]
}
2025-01-31 21:44:06,022:DEBUG:acme.client:Storing nonce: N9Ep_yiT7kj5TJ4m_kZOfKegEOLPuA5ikXYPQ12Te_B-x3B9kS0
2025-01-31 21:44:06,022:INFO:certbot._internal.auth_handler:Performing the following challenges:
2025-01-31 21:44:06,023:INFO:certbot._internal.auth_handler:http-01 challenge for opsforall.net
2025-01-31 21:44:06,023:INFO:certbot._internal.plugins.webroot:Using the webroot path /var/www/certbot for all unmatched domains.
2025-01-31 21:44:06,023:DEBUG:certbot._internal.plugins.webroot:Creating root challenges validation dir at /var/www/certbot/.well-known/acme-challenge
2025-01-31 21:44:06,024:DEBUG:certbot._internal.plugins.webroot:Attempting to save validation to /var/www/certbot/.well-known/acme-challenge/0c0H_DprVNJOEoPqlNXVneWSM-zzr1eGnzPd2nn7VN0
2025-01-31 21:44:06,024:DEBUG:acme.client:JWS payload:
b'{}'
2025-01-31 21:44:06,027:DEBUG:acme.client:Sending POST request to https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902281074/dZlnlA:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS1zdGFnaW5nLXYwMi5hcGkubGV0c2VuY3J5cHQub3JnL2FjbWUvYWNjdC8xODI3ODQzMTQiLCAibm9uY2UiOiAiTjlFcF95aVQ3a2o1VEo0bV9rWk9mS2VnRU9MUHVBNWlrWFlQUTEyVGVfQi14M0I5a1MwIiwgInVybCI6ICJodHRwczovL2FjbWUtc3RhZ2luZy12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2NoYWxsLzE4Mjc4NDMxNC8xNTkwMjI4MTA3NC9kWmxubEEifQ",
  "signature": "O-PoIPsdWFQ5CPX9tIDfcWR-kl11zEl5LHGKWLk9bTPlQlwcfOD3bKn0np8zXfRODthGeXnsP_-dxgElKgRJMCZ5JV78v9VAgxRS5kV2FeqSdD3H749u-1pPqyiCE_Yq8dvAjWAQR1nByc2cRU-Czs5g4qJfU27O7BO7A7s-6gexY9EtRKYUO3zc5x-HPrKnzMlF2PisWW0LDFboAL2kka_0gKWlC3ekhek9KstWPYOH0B_cWLkkomTbI9HqhpkBDZpMuL2c7mQVF4LVnLrPMx6SF30knTT5_M8zKzuVrOW4IIP5ssRtyC-a_XUr3ShZWPd_4fEslgBxE8ewYSOyFg",
  "payload": "e30"
}
2025-01-31 21:44:06,097:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "POST /acme/chall/182784314/15902281074/dZlnlA HTTP/1.1" 200 201
2025-01-31 21:44:06,097:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 31 Jan 2025 21:44:06 GMT
Content-Type: application/json
Content-Length: 201
Connection: keep-alive
Boulder-Requester: 182784314
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index", <https://acme-staging-v02.api.letsencrypt.org/acme/authz/182784314/15902281074>;rel="up"
Location: https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902281074/dZlnlA
Replay-Nonce: N9Ep_yiTfcHiV3MTAORbVx4n5f7QVAdkXuFVfxEndxQj-vYi88E
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "type": "http-01",
  "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902281074/dZlnlA",
  "status": "pending",
  "token": "0c0H_DprVNJOEoPqlNXVneWSM-zzr1eGnzPd2nn7VN0"
}
2025-01-31 21:44:06,097:DEBUG:acme.client:Storing nonce: N9Ep_yiTfcHiV3MTAORbVx4n5f7QVAdkXuFVfxEndxQj-vYi88E
2025-01-31 21:44:06,098:INFO:certbot._internal.auth_handler:Waiting for verification...
2025-01-31 21:44:07,098:DEBUG:acme.client:JWS payload:
b''
2025-01-31 21:44:07,099:DEBUG:acme.client:Sending POST request to https://acme-staging-v02.api.letsencrypt.org/acme/authz/182784314/15902281074:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS1zdGFnaW5nLXYwMi5hcGkubGV0c2VuY3J5cHQub3JnL2FjbWUvYWNjdC8xODI3ODQzMTQiLCAibm9uY2UiOiAiTjlFcF95aVRmY0hpVjNNVEFPUmJWeDRuNWY3UVZBZGtYdUZWZnhFbmR4UWotdllpODhFIiwgInVybCI6ICJodHRwczovL2FjbWUtc3RhZ2luZy12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LzE4Mjc4NDMxNC8xNTkwMjI4MTA3NCJ9",
  "signature": "316gW77B3sIjKGrVfLNnTK5Fmk2ohoovzANFmVzgGdk6C9Ami-TgXPaL4qN29afjAE0Q8HIy4weVmLmBqDKAo8KWo7fnUDDTp6uQUa6h8AvhmhQ-ulOC3bEXXXKergWnYXb2n8Bp5bkYYsPpjprjgUY3BElmWyMWZDwEPsCdL1gFpiaG6-0NsY-47_nF8jxdtgSkVo2vl6T4dcDUdKuCrjzF4N6gY719vp3nwpyAGE6P-on2-13doBzKcgmOFtQScJgDR-HNahwwquF5sY2hxDX4No6frahw_SjK-NpP-wlWmbqWHAG4UuystjjQ6_azlXiRhMGM1s3nxlueaspsiw",
  "payload": ""
}
2025-01-31 21:44:07,164:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "POST /acme/authz/182784314/15902281074 HTTP/1.1" 200 1264
2025-01-31 21:44:07,164:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 31 Jan 2025 21:44:07 GMT
Content-Type: application/json
Content-Length: 1264
Connection: keep-alive
Boulder-Requester: 182784314
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: xMYr_xatn5f_3R7n4XCwJ0bxQn_r5nOujt2Us6WhVBr8JWhbjJo
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "opsforall.net"
  },
  "status": "invalid",
  "expires": "2025-02-07T21:44:05Z",
  "challenges": [
    {
      "type": "http-01",
      "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902281074/dZlnlA",
      "status": "invalid",
      "validated": "2025-01-31T21:44:06Z",
      "error": {
        "type": "urn:ietf:params:acme:error:unauthorized",
        "detail": "54.226.228.6: Invalid response from http://opsforall.net/.well-known/acme-challenge/0c0H_DprVNJOEoPqlNXVneWSM-zzr1eGnzPd2nn7VN0: \"\u003c!DOCTYPE html\u003e\u003chtml lang=\\\"en\\\"\u003e\u003chead\u003e\\n  \u003cmeta charset=\\\"utf-8\\\"\u003e\\n  \u003ctitle\u003eAngularFrontend\u003c/title\u003e\\n  \u003cbase href=\\\"/\\\"\u003e\\n  \u003cmeta name=\\\"\"",
        "status": 403
      },
      "token": "0c0H_DprVNJOEoPqlNXVneWSM-zzr1eGnzPd2nn7VN0",
      "validationRecord": [
        {
          "url": "http://opsforall.net/.well-known/acme-challenge/0c0H_DprVNJOEoPqlNXVneWSM-zzr1eGnzPd2nn7VN0",
          "hostname": "opsforall.net",
          "port": "80",
          "addressesResolved": [
            "54.226.228.6"
          ],
          "addressUsed": "54.226.228.6"
        }
      ]
    }
  ]
}
2025-01-31 21:44:07,164:DEBUG:acme.client:Storing nonce: xMYr_xatn5f_3R7n4XCwJ0bxQn_r5nOujt2Us6WhVBr8JWhbjJo
2025-01-31 21:44:07,165:INFO:certbot._internal.auth_handler:Challenge failed for domain opsforall.net
2025-01-31 21:44:07,165:INFO:certbot._internal.auth_handler:http-01 challenge for opsforall.net
2025-01-31 21:44:07,165:DEBUG:certbot._internal.display.obj:Notifying user: 
Certbot failed to authenticate some domains (authenticator: webroot). The Certificate Authority reported these problems:
  Domain: opsforall.net
  Type:   unauthorized
  Detail: 54.226.228.6: Invalid response from http://opsforall.net/.well-known/acme-challenge/0c0H_DprVNJOEoPqlNXVneWSM-zzr1eGnzPd2nn7VN0: "<!DOCTYPE html><html lang=\"en\"><head>\n  <meta charset=\"utf-8\">\n  <title>AngularFrontend</title>\n  <base href=\"/\">\n  <meta name=\""

Hint: The Certificate Authority failed to download the temporary challenge files created by Certbot. Ensure that the listed domains serve their content from the provided --webroot-path/-w and that files created there can be downloaded from the internet.

2025-01-31 21:44:07,166:DEBUG:certbot._internal.error_handler:Encountered exception:
Traceback (most recent call last):
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.

2025-01-31 21:44:07,166:DEBUG:certbot._internal.error_handler:Calling registered functions
2025-01-31 21:44:07,166:INFO:certbot._internal.auth_handler:Cleaning up challenges
2025-01-31 21:44:07,166:DEBUG:certbot._internal.plugins.webroot:Removing /var/www/certbot/.well-known/acme-challenge/0c0H_DprVNJOEoPqlNXVneWSM-zzr1eGnzPd2nn7VN0
2025-01-31 21:44:07,167:DEBUG:certbot._internal.plugins.webroot:All challenges cleaned up
2025-01-31 21:44:07,167:DEBUG:certbot._internal.log:Exiting abnormally:
Traceback (most recent call last):
  File "/usr/local/bin/certbot", line 33, in <module>
    sys.exit(load_entry_point('certbot', 'console_scripts', 'certbot')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/main.py", line 19, in main
    return internal_main.main(cli_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1873, in main
    return config.func(config, plugins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1579, in certonly
    lineage = _get_and_save_cert(le_client, config, domains, certname, lineage)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 142, in _get_and_save_cert
    lineage = le_client.obtain_and_enroll_certificate(domains, certname)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 518, in obtain_and_enroll_certificate
    cert, chain, key, _ = self.obtain_certificate(domains)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 429, in obtain_certificate
    orderr = self._get_order_and_authorizations(csr.data, self.config.allow_subset_of_names)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 497, in _get_order_and_authorizations
    authzr = self.auth_handler.handle_authorizations(orderr, self.config, best_effort)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.
2025-01-31 21:44:07,169:ERROR:certbot._internal.log:Some challenges have failed.
